src/utils/valuesConfig.ts

```diff
@@ -0,0 +1,59 @@
+import { DataClass, DataItem } from 'scrivito'
+import { ensureString } from './ensureString'
+
+const values: Partial<Record<string, Partial<Record<string, string[]>>>> = {}
+const localizers: Partial<
+  Record<string, Partial<Record<string, Partial<Record<string, string>>>>>
+> = {}
```

Comment:
This works, but is hard to understand. Could you maybe add more intermediate type/interfaces to help to improve understanding?

Reply:
FYI If we'd enable `noUncheckedIndexedAccess` we could drop the `Partial` part. But maybe this makes other areas of the codebase harder to maintain.